enterprise/server/raft/store/store.go

```diff
@@ -97,7 +102,52 @@ type Store struct {
 	egCancel context.CancelFunc
 }
 
-func New(env environment.Env, rootDir string, nodeHost *dragonboat.NodeHost, gossipManager interfaces.GossipService, sender *sender.Sender, registry registry.NodeRegistry, listener *listener.RaftListener, apiClient *client.APIClient, grpcAddress string, partitions []disk.Partition) (*Store, error) {
+// We need to provide a factory method that creates the DynamicNodeRegistry, and
```

Comment:
registryHolder implements NodeRegistryFactory. ....When nodeHost is created, it will call Create() to create the registry ....